src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -490,7 +485,7 @@ public String getQuestionResultStatisticsCsv(
         return studentResults;
     }
 
-    protected Map<String, TeamEvalResult> getTeamResults(List<String> teamNames,
+    private Map<String, TeamEvalResult> getTeamResults(List<String> teamNames,
```

Comment:
@tarun14110 I see that this line is giving you troubles in PMD. It's a bug on their side, so the `protected` thing is a temporary workaround for it. But there's one way you can improve it: instead of making it protected, add this annotation `@SuppressWarnings("PMD.UnusedPrivateMethod")` on the method. Also, mention that it's a false positive by PMD.